www/docs/usage/commandline.md

```diff
@@ -59,13 +59,32 @@ This can either be set to:
 For example:
 
 ```bash
-export TF_PRODUCT=opentofu
+export TF_PRODUCT="opentofu"
 tfswitch # Will install opentofu instead of terraform
 ```
 
+### `TF_LOG_LEVEL`
+
+`TF_LOG_LEVEL` environment variable can be set to override default log level.
+
+- Supported log levels:
+  - `INFO`:   includes `PANIC`, `FATAL`, `ERROR`
```

Comment:
I guess, INFO debug level also includes `INFO` ;) 

Worth mentioning what the default log level is?